service/storage_proxy.cc

```diff
@@ -380,20 +380,16 @@ class storage_proxy::remote {
 
         std::vector<frozen_mutation_and_schema> mutations;
         auto timeout = *t;
-        {
```

Comment:
OK, scopes are there until re-indent.

Reply:
Looks like Avi cannot handle over-indented code without at least some brackets wrapping it.

Reply:
My brain dumps core if I see mismatched indentations.